skyportal/facility_apis/fake_request.py

```diff
@@ -0,0 +1,33 @@
+from tornado import httputil
+
+
+class FakeApplication(object):
```

Comment:
in class construction Python 3 does not subclass `object`
```suggestion
class FakeApplication:
```